.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
+
+install_status=1
+
+SRC="${PWD}/_out/build_install"
+
+# Expect the host filesystem to be mounted in following path.
+HOST_MOUNT=/host
+
+# Some files are copied over /usr which on Red Hat CoreOS (rhcos) is mounted
+# read-only by default. So re-mount it as read-write, otherwise files won't
+# get copied.
+mount -o remount,rw $HOST_MOUNT/usr
+
+# The host's '/opt' and '/usr/local' are symlinks to, respectively, '/var/opt'
```

Comment:
what about here '/usr/local' ?

Reply:
So, the whole idea about Red Hat CoreOS is to provide the user an immutable system, quite similar to Fedora Silverblue (https://silverblue.fedoraproject.org/) or Fedora CoreOS (https://github.com/coreos/fedora-coreos-tracker).

Those systems do have `/usr` being mounted as read-only. Apart from a few "workarounds" where wouldn't be to problematic for users to override, such as `/opt` or `/usr/local` ... those workarounds consist in having those directories being symlinks to directories under `/var`, which is not read-only.